site/assets/js/color-modes/index.js

```diff
@@ -62,6 +68,13 @@
           localStorage.setItem('theme', theme)
           setTheme(theme)
           showActiveTheme(theme)
+
+          // After changing the theme, hide offcanvas
+          const oc = document.querySelector('[aria-modal][class*=show][class*=offcanvas-]')
+          if (oc !== null) {
+            // eslint-disable-next-line no-undef
```

Comment:
No need to disable this, just specify `bootstrap: false` in globals.

Reply:
I changed it, sorry, the header is index.json. Confused by .eslintrc.json.

Not enough bootstrap: false in globals. There are four issues in 4 files. error  'bootstrap' is already defined as a built-in global variable  no-redeclare. Need the rule:

```json
  "globals": {
    "bootstrap": false
  },
  "rules": {
    "no-redeclare": [ "error", { 
        "builtinGlobals": false
      }
    ],
```

I also added the .eslintrc.json file.